inc/environment_indicator/namespace.php

```diff
@@ -88,6 +89,31 @@ function add_admin_bar_dashboard_link( WP_Admin_Bar $wp_admin_bar ) {
 	] );
 }
 
+/**
+ * Add menu item to the Altis logo in the admin bar for creating a support ticket.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar WP_Admin_Bar instance, passed by reference.
+ */
+function add_admin_bar_support_ticket_link( WP_Admin_Bar $wp_admin_bar ) {
+	if ( ! is_admin_bar_showing() ) {
+		return;
+	}
+
+	$env_name    = get_environment_name();
+	$support_url = 'unknown' !== $env_name ? "https://dashboard.altis-dxp.com/#/support/new?applications[]={$env_name}" : 'https://dashboard.altis-dxp.com/#/support/new';
```

Comment:
This line is super long; can we move the base URL (`.../support/new`) to a constant?

Also, this should use either `sprintf` or `add_query_arg` for the argument, and it needs to be URL encoded.

Reply:
@rmccue thanks.

`add_query_arg` adds the args before the `#` in the URL, so getting `https://dashboard.altis-dxp.com/?applications%5B%5D=test#/support/new`, I guess anchor comes after the args in the URL structure.

As for `sprintf` I remember a feedback that URL should be escaped in full, not parts. I've updated code. Let me know if that works.